ts/Extensions/GeoJSON.ts

```diff
@@ -574,7 +574,11 @@ function geojson(
         let pointOptions: (MapPointOptions|MapPointPointOptions|undefined);
 
         if (
-            (hType === 'map' || hType === 'mapbubble') &&
+            (
+                hType === 'map' ||
+                hType === 'mapbubble' ||
+                hType === 'tiledwebmap'
```

Comment:
I don't think the tiledwebmap should relate to GeoJSON at all. As we can see from your demo, the tiledwebmap works also when we remove the mapData.